ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -126,6 +126,10 @@ literals:
 - outside_air_relative_humidity_sensor
 - outside_air_temperature_sensor
 - outside_air_wetbulb_temperature_sensor
+- outside_air_drybulb_temperature_sensor
+- wind_speed_sensor
+- wind_direction_angle_sensor
+- outside_air_water_precipitation_sensor
```

Comment:
`outside_air_water_precipitation_level_sensor` is needed here, since precipitation is more of a descriptor than a measurement. level already exists as a measurement.

Reply:
Thanks Done the changes